### ISSUE TITLE
Feature request: private torrent support

### ISSUE THREAD

**Issue by pgeri91** [steeve/plugin.video.pulsar#46]
This is a great addon, but it would be greater if it supported private torrents. Thanks for your work so far.

**Comment by realgam3** [comment]
agree, using magneturi isn't possible in some trackers (torrent sites) and pulsar translate torrent files to magneturis.
